src/main/appengine/app.yaml

```diff
@@ -1,2 +1,3 @@
 runtime: java11
-includes: ${HOME}/team_41/local/env_variables.yaml
\ No newline at end of file
+includes:
+  - /home/jhernandez/team_41/local/env_variables.yaml
```

Comment:
I tried to generalize the address with `${USER}` OR `${HOME}` but those env vars are not recognized; creating an error. This will make it such we would need to manually change the address when anyone else tries to deploy.

Reply:
Thanks for figuring this out!